components/filter/README.md

```diff
@@ -149,6 +149,24 @@ The filter will announce changes to filter selections, search results, and when
 
 <!-- docs: end hidden content -->
 
+### Troubleshooting
```

Comment:
Should we straight-up recommend using `repeat` any time they're iterating over dimensions or values using a `forEach`? Then you could call this section "Iterating with forEach" or something like that.